examples/.rayproject/project.yaml

```diff
@@ -0,0 +1,23 @@
+# This file is generated by `ray project create`.
+
+name: ray-tutorial-examples
+
+description: "The Ray tutorial examples"
+tags: ["ray-tutorial", "examples", "newsreader", "map-reduce", "modin"]
```

Comment:
nice! what does `tags` do?

Reply:
Right now it is just metadata, but can later be displayed in the web UI and used to discover projects :)